Rubberduck.Parsing/Annotations/Concrete/DefaultMemberAnnotation.cs

```diff
@@ -11,8 +12,8 @@ namespace Rubberduck.Parsing.Annotations
     public sealed class DefaultMemberAnnotation : FixedAttributeValueAnnotationBase
     {
         public DefaultMemberAnnotation()
-            : base("DefaultMember", AnnotationTarget.Member, "VB_UserMemId", new[] { "0" })
+            : base("DefaultMember", AnnotationTarget.Member, "VB_UserMemId", new[] { WellKnownDispIds.Value.ToString() })
```

Comment:
Note: if `WellKnownDispIds` were an `enum`, that string would be "`Value`" - but it's a `const` so, this works.

Reply:
ToString() seems like a function that can be pretty dangerous for people learning the language?

Reply:
By default (any class or struct can override it), `ToString` returns the name of the type. For `enum` types it returns the name of the member; to get the underlying member value you'd need to cast it to an `int` first, and then invoke `ToString` against that integer value.

It's not dangerous, just.. the kind of things that bite you *once* ;-)